documentation/homework/week1/src/components/todoAdd.js

```diff
@@ -0,0 +1,45 @@
+import React,{Component} from 'react';
+
+class TodoAdd extends Component{
+    state = {
+        id:'',
+        description:'',
+        deadline:'',
+        done:false
+    }
+    onChangeState(field,value){
+        const currentState=this.state;
+        currentState[field]=value;
+        this.setState(currentState);
+    }
+    onAddFunction(id,description,deadline,done){
```

Comment:
It's not necessary to write "function" in your function names. We already know it's a function! Also, please add spaces before the parentheses and between the arguments (after the commas).